src/libcpu/src/jit/jit_loadstore.cpp

```diff
@@ -88,8 +88,13 @@ loadGeneric(PPCEmuAssembler& a, Instruction instr)
          a.movddup(dst, dst);  // Copy to ps1 as well
       } else {
          decaf_check(sizeof(Type) == 8);
-         auto dst = a.loadXmmRegisterReadWrite(a.fprps[instr.rD]);
```

Comment:
load____ReadWrite indicates that the register will be both read and written.  You should be still using ReadWrite, even if you copy out to a temporary before you use it, though I think there is another solution we should be using...